aptos/contracts/sources/event.move

```diff
@@ -0,0 +1,39 @@
+module pyth::event {
```

Comment:
Maybe too much for now, can we have events similar to the solidity contract? e.g., have old price, have fee, ...